src/api/app/models/staging/workflow.rb

```diff
@@ -82,6 +82,10 @@ def self.load_users(staging_projects)
     users_hash
   end
 
+  def autocomplete(num)
+    unassigned_requests.where('CAST(bs_requests.id AS CHAR) LIKE ?', "%#{num}%")
```

Comment:
I have a concern, regarding this. Here you are searching by bs_request.id, and in the [controller](https://github.com/openSUSE/open-build-service/pull/9478/files#diff-b778c1c1eeae49afc99a16fa4a269f14R59) you returns the number of the requests that could or not the same as the request number.

Reply:
updated, could you please take a look @DavidKang ?